servercfg/serverconf.go

```diff
@@ -249,7 +252,12 @@ func GetMessageQueueEndpoint() (string, bool) {
 	} else {
 		host = "ws://" + host
 	}
-	return host + ":" + GetMQServerPort(), secure
+	host += ":" + GetMQServerPort()
+	// websocket listen endpoint for EMQX broker is ws://host:port/mqtt
+	if GetBrokerType() == EmqxBrokerType {
+		host += "/mqtt"
```

Comment:
The /mqtt should just be part of the config/env variable. The user can change the path on their instance if they wish.

Reply:
@0xdcarns this is just for the netmaker internal connection to the broker. What do you mean by changing the path on their instance ?

Reply:
when they specify BROKER env variable it should be something like `broker.mydomain.com` (for mosquitto) or `broker.mydomain.com/mqtt` (for emqx), then netmaker just consumes it and passes it on to clients

Reply:
but then we would have to split it anyways and then append MQ_PORT to the domain and then add the `/mqtt` part at the end

Reply:
@0xdcarns done

Reply:
was decided to use a full endpoint, maybe a better env variable is in order (BROKER_ENDPOINT)

Reply:
@0xdcarns done